### ISSUE TITLE
Migration

### ISSUE THREAD

**Issue by inf3rno** [inf3rno/e2e#114]
I decided to rename the repo and the NPM package as well.

The new name is "e2e" and the new package name is "@inf3rno/e2e". The "e2e-testing" was never really registered, I registered the "@inf3rno/e2e-testing" instead. My mistake. At least I now realized that it is better to move all my old npm packages under that scope, or just deprecate them.

**Comment by inf3rno** [comment]
I have another project I tried to migrate and start from 1.0 again. I wanted to remove the .git directory of that project, and accidentally deleted the .git of this one. So I don't know now what changed recently. Lucky me, I just installed a backup tool, so I can check the difference between the 2 directories and decide whether it's worth to copy them, or just write it again.

**Comment by inf3rno** [comment]
I guess waking up at 3:30 AM is not a good practice. Now that it is 4:30 PM, I am getting tired and making silly mistakes.

**Comment by inf3rno** [comment]
Does not work, the diff is based on file dates, not on checksums in this app.

**Comment by inf3rno** [comment]
I need to check github issues activity. I don't think I closed any issue without pushed commit, but I have to be sure.

**Comment by inf3rno** [comment]
Ok, I checked the closed issues. There weren't any which needed commits, so it is ok to continue with the relative old code. As far as I remember I was working on the #109 , but it took a lot of time to investigate the issue.

**Comment by inf3rno** [comment]
Yepp I added a mocha fix, which throws unhandled rejections as errors, because mocha did not fails by unhandled rejections. Simply adding the file was enough. After that I threw an UnexpectedNavigation error from the Promise.resolve() the lib uses. The only problem, that I cannot test for that error, because it is not handled. I figured that the code would be much better without relying on events. So for example I could do `navigator.promise.reject(new UnexpectedNavigation())` and I could do `await navigator.promise` in the test which will throw an error, because the navigation was not expected. That's why I made a small dependency lib: "Promise.exposed". Ok. I can continue this, but first I should rename everthing. I won't publish this with the new name on NPM yet. Maybe later by the 0.2 release, so that is not an issue here.
